src/plugins/opencv/nodes/capture/lytro_lightfield.cpp

```diff
@@ -80,42 +83,30 @@ dependency_graph::State compute(dependency_graph::Values& data) {
 		lightfields::Raw raw;
 		file >> raw;
 
-		width = raw.metadata()["image"]["width"].asInt();
-		height = raw.metadata()["image"]["height"].asInt();
+		meta = raw.metadata();
+
+		// assemble the lightfield pattern
+		pattern = lightfields::Pattern::fromMetadata(meta);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'pattern' is assigned a value that is never used.](https://app.codacy.com/manual/martin.prazak/possumwood/pullRequest?prid=5012266)